app/src/main/res/layout/activity_main.xml

```diff
@@ -39,13 +40,24 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/tvSnippetToken" />
 
+
     <RelativeLayout
         android:id="@+id/fragment_container"
         android:layout_width="0dp"
         android:layout_height="0dp"
         app:layout_constraintTop_toBottomOf="@id/btnCCTokenizer"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"/>
+        app:layout_constraintEnd_toEndOf="parent" />
+
+
+    <androidx.compose.ui.platform.ComposeView
```

Comment:
Wild. You utilize compose so that you don't need XML anymore. Let's stick with it for now. I will refactor it to full compose in the future